src/app/accounts/register.component.html

```diff
@@ -1,45 +1,134 @@
 <div class="layout" [ngClass]="['layout', layout]">
+  <!-- ENTERPRISE 2 Header -->
   <header class="header" *ngIf="layout === 'enterprise2'">
     <div class="container">
       <div class="row">
         <div class="col-7">
           <img
             alt="Bitwarden"
             class="logo mb-2"
-            src="../../images/register-layout/logo-horizontal-white.png"
+            src="../../images/register-layout/logo-horizontal-white.svg"
           />
         </div>
       </div>
     </div>
   </header>
+
+  <!-- TEAMS 1 Header -->
+  <header class="header" *ngIf="layout === 'teams1'">
+    <div class="container">
+      <div class="row">
+        <div class="col-7">
+          <img
+            alt="Bitwarden"
+            class="logo mb-2"
+            src="../../images/register-layout/logo-horizontal-white.svg"
+          />
+        </div>
+      </div>
+    </div>
+  </header>
```

Comment:
This seems to be duplicated for all layouts?

Reply:
Yes. It is duplicated. There is a request for differentiation between them but a final decision hasn't been made. I'll try again to get feedback. Until then, are you suggesting we eliminate the dupes until variations are necessary? I'm fine leaving them until that determination is made, but will defer to your preference.

Reply:
We can postpone it, but we shouldn't merge it before it's resolved.

Reply:
We need to leave the dupes. Freshsales triggers our drip campaigns based on URL params. Unless there's a better way to accomplish this, we'll have to roll with this. We would like to get this resolved as soon as we can so we can make sure the campaign gets tracked properly. Let me know if there is anything else you can see that needs to be addressed. Thanks!

Reply:
Can't we have `<header class="header" *ngIf="layout === 'teams1' || layout === 'enterprise1'">`?

Reply:
I'm not sure we can default to teams1 if layout != enterprise1. I'll ask the demand gen team if that's acceptable.